.eslintrc.json

```diff
@@ -20,7 +20,8 @@
     "no-console": "error",
     "simple-import-sort/imports": "error",
     "unicorn/prevent-abbreviations": "off",
-    "unicorn/no-array-reduce": "off"
+    "unicorn/no-array-reduce": "off",
+    "prettier/prettier": ["error", { "endOfLine": "auto" }, {"usePrettier": true}]
```

Comment:
What is this change for? I've been using `lf` as EOL and changing it to `auto` does not seem to make any effect.
https://prettier.io/docs/en/options.html#end-of-line

Reply:
![image](https://user-images.githubusercontent.com/64137531/205484779-60f8e7d2-dce6-40ba-b7ed-3f456594eb5d.png)

and all files have this error but i remove this line and run a script into packege.json to solve all files